forge-gui/res/cardsfolder/d/discord_lord_of_disharmony.txt

```diff
@@ -0,0 +1,23 @@
+Name:Discord, Lord of Disharmony
+ManaCost:2 B R
+Types:Legendary Creature Chimera
+PT:3/5
+K:Flying
+T:Mode$ Phase | Phase$ End of Turn | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ ChooseName | TriggerDescription$ At the beginning of your end step, choose a random nonland Magic card name. Until your next end step, you may cast a copy of a spell with that name, and mana of any type can be spent to cast it. If you cast a spell this way, copy this ability if NICKNAME is on the battlefield.
+
```

Comment:
While I assume these helped with readability while working on the script, empty lines are only used at the end of scripts and to separate the halves/faces of split/flip/double-faced/specialize cards.

Reply:
Yeah, I saw that seemed to be standard, but wasn't sure if there was a reason for it. I've cleaned them up now.